polaris-for-vscode/server/package.json

```diff
@@ -0,0 +1,19 @@
+{
+	"name": "polaris-for-vscode",
+	"description": "Polaris Design System VSCode extension",
+	"version": "0.0.1",
+	"author": "Polaris",
+	"license": "MIT",
+	"engines": {
+		"node": "*"
+	},
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/Shopify/polaris.git"
+	},
+	"dependencies": {
+		"vscode-languageserver": "^7.0.0",
+		"vscode-languageserver-textdocument": "^1.0.4"
+	},
+	"scripts": {}
+}
```

Comment:
Since, technically, you can start a LSP from other editors. Shouldn't this be popped out of here? Is this a "I'll do this later issue" :D?

I imagine others might want to _eventually_ start the language server by installing it separately. e.g. `npm install polaris-language-server` or something.

Reply:
was definitely an "I'll get to this eventually" issue 😅 It's a good call out, I think we should keep in mind how the language server and the client will be set up once we have an opinion on how this extension will be affected by the monorepo. For now, I'm chill leaving it like this unless there's a specific need for it to be set up for packaging immediately